vllm/config.py

```diff
@@ -256,6 +257,8 @@ def __init__(self,
                    f"'Please instead use `--hf-overrides '{hf_override!r}'`")
             warnings.warn(DeprecationWarning(msg), stacklevel=2)
 
+        self.pull_model_tokenizer_for_s3(model, tokenizer)
```

Comment:
Make it more clear about we only pull models from S3 for certain cases.

```suggestion
        self.maybe_pull_model_tokenizer_for_s3(model, tokenizer)
```

Reply:
Accepting! Good point, thanks!